stl/src/winapisupp.cpp

```diff
@@ -14,11 +14,36 @@
 #pragma warning(disable : 4265) // non-virtual destructor in base class
 #include <wrl/wrappers/corewrappers.h>
 #pragma warning(pop)
-#include <cstdint>
+
+#if !defined(_ONECORE)
+namespace {
+
+    enum wrapKERNEL32Functions {
+#if !defined(_CRT_WINDOWS) && !defined(UNDOCKED_WINDOWS_UCRT)
+        eGetCurrentPackageId,
+#endif // !defined(_CRT_WINDOWS) && !defined(UNDOCKED_WINDOWS_UCRT)
+
+#if _STL_WIN32_WINNT < _WIN32_WINNT_WIN8
+        eGetSystemTimePreciseAsFileTime,
+#endif // _STL_WIN32_WINNT < _WIN32_WINNT_WIN8
+
+        eMaxKernel32Function

```

Comment:
it strikes me as not the best idea to preprocess out enumerators like this without explicitly assigning values. If this is ever passed across ABI boundaries (it does not appear to be now) it might blow up pretty immediately

Pre-existing though

Reply:
Agreed, pre-existing. I believe that the risk is low (as you mentioned, this table and its enumerators aren't part of the ABI surface area - also, the conditions are inherently of the form "how is the STL being built"), and it's further mitigated by this PR, as the table and its enumerators are now confined to an unnamed namespace in a single TU.

That said, it now seems to me that the case for having a table is weaker than ever. We never iterate over the table, so we could simply use separate function pointers (which the compiler could still store densely). That would avoid any concerns about varying enumerator values, and actually make debugging potentially easier. I may explore this in a followup PR.